app/src/components/cards/collectionCard.tsx

```diff
@@ -76,7 +76,9 @@ const CollectionCard = ({
             },
           }}
         >
-          {`${collection.numItems} items`}
+          {collection.numberOfDigitizedItems > 1
```

Comment:
Another common pattern is 
```
`${collection.numberOfDigitizedItems} item${collection.numberOfDigitizedItems > 1 ? "s" : "" }
```

They both give you the same result but if you need to pluralize other words in the future, you could make a function and make the number variable more generic.